pkgs/applications/misc/river-tag-overlay/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromSourcehut, nixos, wayland, pixman, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "river-tag-overlay";
+  version = "1.0.0";
+
+  buildInputs = [ wayland pixman pkg-config ];
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-hLyXdLi/ldvwPJ1oQQsH5wgflQJuXu6vhYw/qdKAV9E=";
+  };
+
+  makeFlags = [
+    "DESTDIR=${placeholder ''out''}"
+    "PREFIX="
+  ];
+
+  meta = with lib; {
+    description = "A pop-up showing tag status";
+    homepage = "https://git.sr.ht/~leon_plickat/lswt";
+    license = licenses.gpl3;
+    maintainers = [ maintainers.edrex ];
```

Comment:
It seems that you need to add yourself to `maintainers-list.nix` file.

Reply:
Done, and I broken out the maintainer-list.nix change into a separate commit that's shared with #158529 as suggested by @urandom2.